lib/ex_aws/cloud_search/document.ex

```diff
@@ -0,0 +1,5 @@
+defprotocol ExAws.CloudSearch.Document do
+  @fallback_to_any true
```

Comment:
Curious about this choice. Are there data types other than the structs that we want to support?